about/index.md

```diff
@@ -1,3 +1,27 @@
 ---
 title: About
 ---
+## What is the Infectious Diseases Toolkit?
+The Infectious Diseases Toolkit (IDTk) was created as part of the [BY-COVID]( https://by-covid.org/) project. It is a community effort to expose best practices and showcase solutions to data challenges affecting the response to infectious diseases outbreaks.
+
+## What are the aims of IDTk?
+The IDTk provides a platform for knowledge exchange to enable data mobilisation, connection & exposure, and use & analysis.
+By providing a compilation of past and present efforts, collecting country specific strategies and distilling contextualised best practices from previous responses, IDTk aims to enhance  infectious diseases outbreak preparedness.
+
+## What does the IDTk offer?
+General guidelines for data handling, analysis and visualisations in the context of infectious diseases. Guidelines are divided in categories and represent the contribution of communities from each domain.
+Tools, standards and overall resources that are contextually presented, including considerations for their need and use. 
+A collection of showcases. These are examples that combine and apply multiple resources (e.g. standards, tools, workflows) in the response to one or more infectious diseases.
+National resources pages in which country specific experiences, examples and resources are shared.
+The IDTk strives to connect users with the appropriate resources. A collection of tools, resources and training is offered, in addition to signposting external resources.
+ 
+ ## Funding acknowledgement
+Please check our [Support page](/about/support) for information on funding and contributing projects.
+ 
+## How to cite the IDTk
```

Comment:
I don't know if this subtitle fits the content

Reply:
I found it a bit excessive given that there is a full page for support. But I can see that a portion of users might expect funding info on an about page. That is why I thought linking towards that page was better than repeating the information there.

Reply:
I am ok with the content of the subheading, but based on the subtitle I would expect an answer in how I should cite IDTk, but instead it talks about licensing and funding :)

Reply:
I misread the comment :facepalm:, sorry

The citing is so far just attached to the url and the project context in which IDTk is being built, as there is no publication or identifier.

We might add a '##License' subheading above the licensing info. Would that solve it?

Reply:
Adding citation box to separate content